src/ngraph/frontend/onnx_import/core/attribute.hpp

```diff
@@ -16,7 +16,11 @@
 
 #pragma once
 
+#ifdef ONNX_ML
```

Comment:
* We would prefer to avoid using `#ifdef` if possible. Perhaps we should just switch to using ONNX-ML wholesale?
* `ONNX_ML` does not seem specific enough as an option name.
* We should add the option to a list of available options in `CMakeLists`

Reply:
1. Customers having prebuilt/custom onnx will encounter issues, if we hard-code either of onnx or onnx_ml. They have to use whatever the header filer they have available with.
2 & 3. I was thinking about it and have patch to create ONNX_ML as an option. However my skills on cmake are limited, so I may need your help if it doesn't follow guidelines.